bucket/covscript.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": "3.3.2",
+    "description": "Covariant Script interpreter",
+    "homepage": "https://covscript.org/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/covscript/covscript/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/covscript/covscript/releases/download/3.3.2/covscript-win64.7z",
+            "hash": "c1b250bbc995982c33f1e33fefd62a794626d131e71c116145b23a1d2bd076f8",
+            "extract_dir": "build_x64"
+        },
+        "32bit": {
+            "url": "https://github.com/covscript/covscript/releases/download/3.3.2/covscript-win32.7z",
+            "hash": "ddc319251260bc3aab35c931c6e7761a1d8499839fb9965655653999c3de0f27",
+            "extract_dir": "build"
+        }
+    },
+    "pre_install": [
```

Comment:
Why do you do this? What is wrong with .bat shims?

![A](https://i.imgur.com/bFdBX4R.gif)

Clean .bat shim:

![image](https://user-images.githubusercontent.com/13260377/76210028-8d1af700-6203-11ea-9616-09aa563fe16e.png)

Reply:
I was wrong. 
*.bat* shims works just fine. Therefore the ps1 script is not needed.